edgelet/edgelet-core/src/settings.rs

```diff
@@ -411,7 +418,15 @@ impl Provisioning {
     }
 
     pub fn dynamic_reprovisioning(&self) -> bool {
-        self.dynamic_reprovisioning
+        // If using DPS provisioning and its always_reprovision_on_startup is false,
```

Comment:
Should there be a test for this?

Reply:
Talked it over with Anca and we decided it's better to not override it at all, just in case there's a bug that triggers spurious reprovisioning.